Realmius.Server/QuickStart/RealmiusServer.cs

```diff
@@ -12,19 +13,26 @@ public class RealmiusServer
         public static void SetupSignalRServer<T>(
             string url,
             IAppBuilder app,
-            IRealmiusServerConfiguration<T> configuration)
+            IRealmiusServerConfiguration<T> configuration,
+            ILogger logger = null)
         {
+            if (logger != null)
+            {
+                ChangeTrackingDbContext.Logger = logger;
```

Comment:
it's better to put logger in IRealmiusServerConfiguration, rather than in ChangeTrackingDbContext.
moreover, logger should not be static (as it currently is)